src/taipy/core/cycle/_cycle_manager.py

```diff
@@ -84,12 +86,12 @@ def _get_end_date_of_cycle(frequency: Frequency, start_date: datetime):
     @classmethod
     def _hard_delete(cls, cycle_id: CycleId):
         cycle = cls._get(cycle_id)
-        entity_ids_to_delete = cls._get_owned_entities(cycle)
+        entity_ids_to_delete = cls._get_owned_entity_ids(cycle)
         entity_ids_to_delete.cycle_ids.add(cycle.id)
         cls._delete_entities_of_multiple_types(entity_ids_to_delete)
 
     @classmethod
-    def _get_owned_entities(cls, cycle: Cycle) -> _EntityIds:
+    def _get_owned_entity_ids(cls, cycle: Cycle) -> _EntityIds:
```

Comment:
```suggestion
    def _get_children_entity_ids(cls, cycle: Cycle) -> _EntityIds:
```

Reply:
Yes, I agree. The 'children' naming is more consistent with the new naming convention you proposed, @toan-quach